examples/lib/stories/experimental/layout_component_example.dart

```diff
@@ -0,0 +1,106 @@
+import 'dart:async';
+
+import 'package:flame/components.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+import 'package:flutter/material.dart';
+
+class LayoutComponentExample extends FlameGame {
+  static const String description = '''
+    Press the row of buttons after MainAxisAlignment, CrossAxisAlignment,
+    and Gap. They will update the layout of the example row and column
+    layouts to reflect the chosen values.
```

Comment:
When doing ''' blocks there shouldn't be any indentation, because that will be included in the string.